package.json

```diff
@@ -72,7 +72,7 @@
     "build:version": "node scripts/version",
     "build:l10n": "cp node_modules/l20n/dist/web/l20n.min.js public",
     "dev": "npm run build && npm start",
-    "format": "prettier '{frontend/src/,scripts/,server/,test/**/}*.js' 'public/*.css' --single-quote --write",
+    "format": "prettier '{frontend/src/,scripts/,server/,test/**/!(bundle)}*.js' 'public/*.css' --single-quote --write",
```

Comment:
Excluding the Browserified /test/frontend/bundle.js should cut a few seconds off the `npm run format` script.